web/crux-ui/src/components/projects/versions/deployments/add-deployment-card.tsx

```diff
@@ -75,6 +69,15 @@ const AddDeploymentCard = (props: AddDeploymentCardProps) => {
     },
   })
 
+  useEffect(() => {
+    if (nodes && nodes.length < 1) {
+      toast.error(t('nodeRequired'))
+    }
+    if (nodes?.length === 1 && !formik.values.nodeId) {
```

Comment:
i think we can always select the first one.

Reply:
We've talked about this with @polaroi8d and we think that if there's more than one node we shouldn't select anything so the user has to.

Reply:
I see. Alright.